### ISSUE TITLE
Error in catching the end of annotations when code examples are in the documentation

### ISSUE THREAD

**Issue by dietmarw** [modelica-tools/atom-language-modelica#17]
Here is an example where the current syntax highlighter currently fails:

``` modelica
model test
  annotation (Documentation(info="<html>
 <pre>
 A = [1,2,3;
   3,4,5;
   2,1,4];
 b = {10,22,12};
 x = Matrices.solve(A,b);
 Matrices.eigenValues(A);
 </pre>
</html>"));
end test;
```

is displayed in atom.io as:
![screenshot from 2015-10-15 19 02 33](https://cloud.githubusercontent.com/assets/9332/10521291/51546922-736f-11e5-8a58-16aa4d26dded.png)

**Comment by tshort** [comment]
This one's fixed in Borris' new grammar that we have the pull request for (#13). Toggling of annotations for this case is probably still broken. (I have one idea for how to fix it.)

**Comment by thorade** [comment]
Fixed by PR #13
If not, feel free to reopen.

**Comment by dietmarw** [comment]
Fix confirmed.
